ztp/source-crs/validatorCRs/informDuValidator.yaml

```diff
@@ -59,63 +27,7 @@ status:
   syncStatus: Succeeded
 ---
 apiVersion: apps/v1
-kind: Deployment
+kind: DaemonSet
```

Comment:
Is it necessary to check status.numberMisscheduled=0 as well?

Reply:
We could add that maybe - Right now we're just checking that the daemonset has been created, not that it has been scheduled.

Originally I tried to check that the `numberUnavailable` was zero, but it seems like instead of setting that to 0 it just removes the field, and the policy never goes compliant.  I'm not sure numberMisscheduled is right to check - That counts pods that are running but on the wrong node...  It might more correct to check that the `numberReady`:and `numberActive` are equal to `desiredNumberScheduled` - This is a bit annoying to do with ACM though.

I think just checking if the daemonset is present is good enough for us to claim the DU profile is "done".